features/fixtures/refract.sourcemap.json

```diff
@@ -334,6 +334,19 @@
                           "content": "<resource name>"
                         }
                       },
+                      "attributes": {
+                        "sourceMap": [
+                          {
+                            "element": "sourceMap",
+                            "content": [
+                              [
+                                173,
+                                20
+                              ]
+                            ]
+                          }
+                        ]
+                      },
```

Comment:
This is a source map for the `object` keyword in `+ Attributes (object)`.  I don't think we need a source map here.